iree/compiler/Dialect/HAL/IR/LoweringConfig.h

```diff
@@ -59,6 +59,13 @@ IREE::HAL::TranslationInfo buildTranslationInfo(
 IREE::HAL::TranslationInfo getTranslationInfo(
     IREE::HAL::ExecutableEntryPointOp entryPointOp);
 
+/// Get the pass pipeline specified in the `translationInfo`
+inline Optional<IREE::HAL::DispatchLoweringPassPipeline>
+getLoweringPassPipeline(IREE::HAL::TranslationInfo translationInfo) {
+  return IREE::HAL::symbolizeEnum<IREE::HAL::DispatchLoweringPassPipeline>(
```

Comment:
There should be a `symbolizeDispatchLoweringPassPipeline` autogenerated that you can use I think.

Reply:
Sure. Just seems like a redirect though....

Reply:
actually it would have been convenient if there was also had a `StringAttr` overload.